consumer_group.go

```diff
@@ -513,6 +513,9 @@ type ConsumerGroupSession interface {
 	// message twice, and your processing should ideally be idempotent.
 	MarkOffset(topic string, partition int32, offset int64, metadata string)
 
+	// Commit the offset to the backend
+	Commit()
```

Comment:
Do we want to call out that this is a blocking sync operation, or will that just be expected?